installer/charts/rhtap-subscriptions/templates/delete_tekton_config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: delete-tekton-config-if-exists
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
I feel it should be a `pre-install` hook of the `tssc-infrastructure`, and the job/SA should be in `installer/charts/tssc-infrastructure/templates/openshift-pipelines/tektonconfig/`.

Reply:
I thought the same, and I tried it, but pre-install hooks run before the templates are there and it will be missing the RBAC to be able to delete.

Reply:
Understood. Can you at least move the files into an `openshift-pipelines` subdir?

Reply:
sounds good

Reply:
done